.evergreen.yml

```diff
@@ -154,6 +168,15 @@ tasks:
     - func: attach-cypress-screenshots
     - func: attach-cypress-videos
 
+  - name: storybook
+    commands:
+    - func: yarn-storybook
```

Comment:
Would it be possible to add a step to attach Storybook files to the task? We do this [in Spruce](https://github.com/evergreen-ci/spruce/blob/92d42b3794307d0f034944a7a806898342cc720a/.evergreen.yml#L234-L246)!